consensus/dummy/dynamic_fees.go

```diff
@@ -23,15 +23,16 @@ func CalcBaseFee(config *params.ChainConfig, feeConfig commontype.FeeConfig, par
 	// If the current block is the first EIP-1559 block, or it is the genesis block
 	// return the initial slice and initial base fee.
 	isSubnetEVM := config.IsSubnetEVM(parent.Time)
-	extraDataSize := params.ExtraDataSize
+	extraDataSize := params.DynamicFeeExtraDataSize
 
 	if !isSubnetEVM || parent.Number.Cmp(common.Big0) == 0 {
 		initialSlice := make([]byte, extraDataSize)
 		return initialSlice, feeConfig.MinBaseFee, nil
 	}
-	if len(parent.Extra) != extraDataSize {
-		return nil, nil, fmt.Errorf("expected length of parent extra data to be %d, but found %d", extraDataSize, len(parent.Extra))
+	if len(parent.Extra) < extraDataSize {
```

Comment:
same comment about doing this only for the Dupgrade

Reply:
This is just loosening the condition.

Previously, it was required to be identical. Now it can be >= 80 bytes since it requires those 80 bytes for the dynamic fee window.

It's ok to loosen the condition here because we verify the exact match prior to DUpgrade activation in `verifyHeader`